prometheus/auto.d/ReplicaOS.yml.example

```diff
@@ -0,0 +1,4 @@
+- targets: [ "10.123.202.13:9100", "10.123.202.13:9187" ]
```

Comment:
Need to remove postgres exporter target from here and change name to "Replica"

Reply:
Fixed in https://github.com/CrunchyData/pgmonitor/pull/89/commits/2cf98aa2d8ae00d6c8c10707d86194af6cc0adf7